roles/kafka_broker/tasks/dynamic_groups.yml

```diff
@@ -39,7 +39,8 @@
   when: installation_method == "archive"
   tags:
     - systemd
-# TODO When Zookeeper is removed, use kafka-metadata-quorum script. See KIP-595
+
+# In Kraft mode we don't need to fetch Controller Broker ID. Any node with roles=broker can't be a controller
```

Comment:
I would double check on this with the kafka team. I think there is an "active controller" amongst the brokers when you are running zk mode or kraft mode. thats a separate thing than kraft controllers.

Reply:
I guess this comment answers the question you've added above, we'll confirm with the kafka team